api/app/Policies/UserPolicy.php

```diff
@@ -92,62 +90,46 @@ public function update(User $user, User $model, array $injected)
      */
     public function delete(User $user, User $model)
     {
-        if ($user->isAdmin()) {
-            if ($user->id !== $model->id) {
-                return true;
-            } else {
-                return false; // Do not allow user to delete their own model.
-            }
-        } else {
-            return false;
-        }
+        return $user->isAbleTo('delete-any-user') && $user->id !== $model->id; // Do not allow user to delete their own model.
     }
 
+    /*******************  APPLICANT QUERIES  *******************/
+
     /**
-     * Determine whether the user can restore the model.
+     * Determine whether the user can view any models.
      *
      * @param  \App\Models\User  $user
-     * @param  \App\Models\User  $model
      * @return \Illuminate\Auth\Access\Response|bool
      */
-    public function restore(User $user, User $model)
+    public function viewAnyApplicants(User $user)
     {
-        return $user->isAdmin();
+        return $user->isAbleTo('view-any-applicantProfile');
     }
 
     /**
-     * Determine whether the user can permanently delete the model.
+     * Determine whether the user can view the model.
      *
      * @param  \App\Models\User  $user
      * @param  \App\Models\User  $model
      * @return \Illuminate\Auth\Access\Response|bool
      */
-    public function forceDelete(User $user, User $model)
-    {
-        return false;
-    }
-
-    /*******************  APPLICANT QUERIES  *******************/
-    /**
-     * Determine whether the user can view any models.
-     *
-     * @param  \App\Models\User|null  $user
-     * @return \Illuminate\Auth\Access\Response|bool
-     */
-    public function viewAnyApplicants(User $user = null)
+    public function viewApplicant(User $user, User $model)
     {
-        return $user->isAdmin();
+        return $user->isAbleTo('view-any-applicantProfile')
+            || (
+                $user->isAbleTo('view-team-applicantProfile')
+                && $this->applicantHasAppliedToPoolInTeams($model, $user->rolesTeams()->pluck('id')
+            )
+        );
     }
```

Comment:
this doesn't allow an applicant to view themselves
which is needed for queries such as 

![image](https://user-images.githubusercontent.com/40485260/224452073-2c6d0a0d-b264-4d38-8d9d-915d0ad741c3.png)

resulting in an inability to access experiences page, and therefore not able to complete applications either
used in 

![image](https://user-images.githubusercontent.com/40485260/224452127-65a2a998-58f9-4693-b46e-f1d7f5bbdcf9.png)

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5858/commits/58d765d99a3f7be0df977e32c8900357b121170f